dashboard-widgets.el

```diff
@@ -211,14 +208,14 @@ Default value is `official', it displays the Emacs logo.  `logo' displays Emacs
 alternative logo.  An integer value is the index of text banner.  A string
 value must be a path to a .PNG or .TXT file.  If the value is nil then no banner
 is displayed."
-  :type '(choice (const  :tag "offical"   official)
-                 (const  :tag "logo"      logo)
-                 (string :tag "a png or txt path"))
-  :group 'dashboard)
-
-(defcustom dashboard-buffer-last-width nil
-  "Previous width of dashboard-buffer."
-  :type  'integer
+  :type '(choice (const   :tag "no banner" nil)
+                 (const   :tag "offical"   official)
+                 (const   :tag "logo"      logo)
+                 (integer :tag "index of a text banner")
+                 (string  :tag "a path to an image or text banner")
+                 (cons    :tag "an image and text banner"
```

Comment:
This is a new option, Should we add an example on the `README` file?

Reply:
done